content/doc/book/using/remote-access-api.adoc

```diff
@@ -16,13 +16,13 @@ single entry point for all features, and instead they are available
 under the `+".../api/"+` URL where `+"..."+` portion is the data that it
 acts on.
 
-For example, if your Jenkins installation sits at http://ci.jruby.org/,
-visiting http://ci.jruby.org/api/ will show just the top-level API
+For example, if your Jenkins installation sits at https://ci.jenkins.io,
+visiting https://ci.jenkins.io/api/ will show just the top-level API
 features available – primarily a listing of the configured jobs for this
 Jenkins instance. +
 Or if you want to access information about a particular build, e.g.
-http://ci.jruby.org/job/jruby-base/lastSuccessfulBuild/, then go to
-http://ci.jruby.org/job/jruby-base/lastSuccessfulBuild/api/ and you'll
+https://ci.jenkins.io/job/jruby-base/lastSuccessfulBuild/, then go to
```

Comment:
I think this should either be non-interactive links to a fictional server `https://ci.example.com` or working links to  a publicly available job, e.g.
https://ci.jenkins.io/job/Core/job/jenkins/job/master/lastSuccessfulBuild/
https://ci.jenkins.io/job/Core/job/jenkins/job/master/lastSuccessfulBuild/api

Reply:
Thanks @zbynek, I've replaced the links